src/gen_dragonfruit/gd_dev_adafruit_MPL115A2_press.h

```diff
@@ -0,0 +1,7 @@
+#include "Adafruit_MPL115A2.h"
+
+#ifndef _PIN_GD_ADAFRUIT_MPL115A2_PRESS_H
+#define _PIN_GD_ADAFRUIT_MPL115A2_PRESS_H
```

Comment:
`_PIN` should be reserved for macros that define pin numbers on the device. This macro is being used as an include guard and shouldn't have `_PIN` pre-pended.